### PR TITLE
ROM integration

### PR DESCRIPTION
**Pull request by flash-gordon**
I had to do a couple of minor changes in order to integrate dry-initializer with rom. The changes in `inherited` hook deserve a test, so I'll add one later

### CODE REVIEW

**Comment by flash-gordon** [comment]
Ok, I rebased this branch on https://github.com/dry-rb/dry-initializer/pull/20

**Comment by nepalez** [comment]
Yeah, I found this clean and pretty, but will wait for your spec before merging

File: lib/dry/initializer/mixin.rb [hunk 1]
```diff
@@ -49,6 +49,8 @@ def initializer_builder
 
     def inherited(klass)
       klass.instance_variable_set :@initializer_builder, initializer_builder.dup
+
+      super(klass)
```
Comment (solnic) [inline lib/dry/initializer/mixin.rb]: `super` will be enough

**Comment by flash-gordon** [comment]
I added a spec

**Comment by nepalez** [comment]
Thank you!

**Comment by solnic** [comment]
Fantastic! Thanks for helping with this

**Pull request merged**